src/Dex.tsx

```diff
@@ -60,9 +83,13 @@ interface DexProps {
 }
 
 function Dex(props: DexProps) {
-  const { pkmn, search, updateSearch, updateCurrentPage, currentPage } = props;
+  const { changeTab,updateType0, updateType1, updateType2, pkmn, search, updateSearch, updateCurrentPage, currentPage } = props;
   const searchInput = <Search search={search} updateSearch={updateSearch} />;
   const emptyState = <p className="silver f1 b tc m0">no pokémon found</p>;
+  gChangeTab = changeTab;
+  gupdateType0 = updateType0;
+  gupdateType1 = updateType1;
+  gupdateType2 = updateType2;
```

Comment:
I'm not really sure what this assignment is for.

Reply:
see comment above